app/assets/javascripts/base.js.erb

```diff
@@ -1,3 +1,5 @@
+//= require jquery
+//= require jquery_ujs
```

Comment:
Is there a way to do this without UJS? We purposefully don't include it as we don't use it around the website, neither jQuery